spec/lib/task-data/schemas/generate-tag-spec.js

```diff
@@ -0,0 +1,33 @@
+// Copyright 2016, EMC, Inc.
+/* jshint node: true */
+
+'use strict';
+
+describe(require('path').basename(__filename), function() {
+    var schemaFileName = 'generate-tag.json';
+
+    var canonical = {
+        "nodeIds": []
+    };
+
+    var negativeSetParam = {
+        "nodeIds": [ "test", 1234 ]
```

Comment:
There is a little tricky here, you need use the Array's Array here, as the testing will create a test case for each element in the array, rather than use the array as a whole. See the code: https://github.com/RackHD/on-tasks/blob/master/spec/lib/task-data/schemas/schema-ut-helper.js#L281,
```
"nodeIds": [  ["test", 123] ]
```